crates/holochain_sqlite/src/sql/p2p_agent_store/extrapolated_coverage.sql

```diff
@@ -0,0 +1,45 @@
+SELECT
+  SUM(
+    -- first, sum up the 0.0 - 1.0 coverage of everyone centered in our arc
+    CASE
+      -- if start is before end
+      WHEN (storage_start_loc <= storage_end_loc) THEN IFNULL(
+        CAST(storage_end_loc AS FLOAT) - CAST(storage_start_loc AS FLOAT),
+        0.0
+      ) / 4294967295.0
+      ELSE -- else if start is after end
+      IFNULL(
+        4294967295.0 - CAST(storage_start_loc AS FLOAT) + CAST(storage_end_loc AS FLOAT),
+        0.0
+      ) / 4294967295.0
+    END
+  ) * (
+    -- then extrapolate assuming similar coverage for the rest of the arc
+    4294967295.0 / CASE
+      WHEN (:start_loc <= :end_loc) THEN -- if start is before end
+      CAST(:end_loc AS FLOAT) - CAST(:start_loc AS FLOAT)
+      ELSE -- else if start is after end
+      4294967295.0 - CAST(:start_loc AS FLOAT) + CAST(:end_loc AS FLOAT)
+    END
+  ) AS coverage
```

Comment:
This formula can be written `(sum_of_peer_arclens / dht_size) * (dht_size / my_arclen)`

So the dht_size term (`4294967295.0`) cancels out and I believe this can be simplified a bit. Because SQLite is using i64 for numbers, so the sum won't overflow, right?

```suggestion
  SUM(
    -- first, sum up the 0.0 - 1.0 coverage of everyone centered in our arc
    CASE
      -- if start is before end
      WHEN (storage_start_loc <= storage_end_loc) THEN IFNULL(
        CAST(storage_end_loc AS FLOAT) - CAST(storage_start_loc AS FLOAT),
        0.0
      )
      ELSE -- else if start is after end
      IFNULL(
        4294967295.0 - CAST(storage_start_loc AS FLOAT) + CAST(storage_end_loc AS FLOAT),
        0.0
      )
    END
  ) / (
    -- then extrapolate assuming similar coverage for the rest of the arc
    CASE
      WHEN (:start_loc <= :end_loc) THEN -- if start is before end
      CAST(:end_loc AS FLOAT) - CAST(:start_loc AS FLOAT)
      ELSE -- else if start is after end
      4294967295.0 - CAST(:start_loc AS FLOAT) + CAST(:end_loc AS FLOAT)
    END
  ) AS coverage
```